apps/web/Containerfile

```diff
@@ -59,6 +62,6 @@ COPY --from=installer /app/apps/web/package.json .
 # https://nextjs.org/docs/advanced-features/output-file-tracing
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/standalone ./
 COPY --from=installer --chown=nextjs:nodejs /app/apps/web/.next/static ./apps/web/.next/static
-COPY --from=installer --chown=nextjs:nodejs /app/apps/web/public ./apps/web/public
+# COPY --from=installer --chown=nextjs:nodejs /app/apps/web/public ./apps/web/public
```

Comment:
AFAICT the `./apps/web/public/` dir doesn't exist and doesn't need to exist: the app works well without it.